src/client/Microsoft.Identity.Client/Platforms/Features/WamBroker/WamBroker.cs

```diff
@@ -160,6 +160,8 @@ internal class WamBroker : IBroker
                     authenticationRequestParameters.LoginHint,
                     authenticationRequestParameters.AppConfig.ClientId).ConfigureAwait(false);
 
+                _logger.Verbose("[WamBroker] Account information provided but not matching account was found ");
```

Comment:
Should this be lower, like Line 183?
nit:
```suggestion
                _logger.Verbose("[WamBroker] Account information provided but no matching account was found.");
```